std_msgs/String pointcloud_topic
std_msgs/String image_topic
---
std_msgs/Empty finished